zntrack/descriptor/descriptor.py

```diff
@@ -0,0 +1,62 @@
+import dataclasses
+import logging
+
+log = logging.getLogger(__name__)
+
+
+@dataclasses.dataclass
+class Metadata:
+    dvc_option: str
+    zntrack_type: str
+
+    @property
+    def dvc_args(self):
+        return self.dvc_option.replace("_", "-")
+
+
+class Descriptor:
```

Comment:
I find it hard to know exactly what a descriptor is as it seems specific to some context. Having a description here or in the header doc string (which is missing) would be very helpful.

Reply:
It is a Python standard term https://docs.python.org/3/howto/descriptor.html

Reply:
I added some in 8bf456b